typescript/appconfig-hosted-configuration-lambda-extension/README.md

```diff
@@ -0,0 +1,58 @@
+# AWS AppConfig Hosted Configuration with Lambda Extension
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples does is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
+
+---
+<!--END STABILITY BANNER-->
+
+This an example of AppConfig fature toggle use case with a hosted configuration and a consuming Lambda using AppConfig Lambda extension integration.
+
```

Comment:
An explanation of the purpose of the lambda would also be important here. At the moment it is very difficult to decipher (and I imagine even more so for someone with less context than me).